idm/idm.go

```diff
@@ -56,7 +56,7 @@ func (i *Idm) GetSpecificID(id uint64) error {
 }
 
 // GetIDInRange returns the first available id in the set within a [start,end] range
```

Comment:
I don't think that this is anymore accurate, correct? (being a comment it's not critical, can be handled on top)